Mlem/App/Views/Shared/Images/Wrappers/ThumbnailImageView.swift

```diff
@@ -8,6 +8,7 @@
 import Foundation
 import MlemMiddleware
 import SwiftUI
+import _QuickLook_SwiftUI // https://github.com/swiftlang/swift/issues/73048
```

Comment:
I believe frameworks formatted like `_QuickLook_SwiftUI` are imported automatically if you import both `QuickLook` and `SwiftUI`. Would importing `QuickLook` here instead also work?

```suggestion
import QuickLook
```